src/click/_termui_impl.py

```diff
@@ -424,26 +424,33 @@ def _pipepager(generator: cabc.Iterable[str], cmd: str, color: bool | None) -> N
                 text = strip_ansi(text)
 
             stdin.write(text.encode(encoding, "replace"))
-    except (OSError, KeyboardInterrupt):
```

Comment:
I'm not sure why OSError has been captured here; I think it's probably better to remove it and re-introduce it with a comment or a more specific solution if it causes a problem.

Reply:
`OSError` used to be `IOError` and it's in since the beginning (aef16506b5da26684103db2e47346dbf49a7e7a1).

If writing files for some reason, I'd like to see an error message, rather than swallowing any error that might occur – but I'm happy to revert this change if the reviewer disagrees or there is a reason I'm not aware of

Reply:
most likely this is done for less noisy behavior in broken pipes or broken SSH sessions?

Reply:
Right, now I also ran into a case. Pushed a version now that ignores `BrokenPipeError` specifically